jpa/boot-jpa-jooq-sample/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM eclipse-temurin:21.0.5_11-jre-alpine as builder
+WORKDIR application
+ARG JAR_FILE=target/batch-boot-jpa-0.0.1-SNAPSHOT.jar
+COPY ${JAR_FILE} application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+# the second stage of our build will copy the extracted layers
+FROM eclipse-temurin:21.0.5_11-jre-alpine
+WORKDIR application
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)